canonical-web-and-design/vanilla-framework
# Add .circleci/config.yml

## Done

Use Circle CI to run Percy instead of Travis

## QA

- check PR status of circle-ci percy job
- Percy screenshots should be generated, status check should pass
- travis check will not work, because it was removed
